src/app/layout.tsx

```diff
@@ -2,6 +2,8 @@ import type { Metadata } from 'next';
 import { Rubik } from 'next/font/google';
 
 import './globals.css';
```

Comment:
https://github.com/import-js/eslint-plugin-import/issues/1084

The import linter ignores unassigned imports while sorting. If you want, you can move this to line 7 to keep it alongside the other sibling imports